build/secondary/third_party/ocmock/BUILD.gn

```diff
@@ -0,0 +1,94 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+assert(is_ios)
+
+ocmock_path = "../../../../third_party/ocmock/Source"
```

Comment:
`//` indicates starting from the buildroot. So just `//third_party/ocmock/Source` would suffice instead of counting up the number of directories to the root.

Reply:
done